### PR TITLE
add note.md and text

### PR DESCRIPTION
**Pull request by DavevonH**

### CODE REVIEW

File: note.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Basic Git Workflow mit GitHub :github:
+
+Once
+In the shell
+git init # create a repo (once per project)
+# don't forget to add a .gitignore file
+git remote add origin <ssh-path-to-github>
```
Comment (TobiasSteinhagen) [inline note.md]: change html in text

**Comment by TobiasSteinhagen** [comment]
change html in text

File: note.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+For each feature
+In the shell
+git switch -c <branchname> # create a new branch
```
Comment (Pia-Lueling) [inline note.md]: LGTM 😌

**Review by Pia-Lueling — COMMENTED** [review]
Perfect :)

**Comment by bermyu** [comment]
good job 👏🏻

**Pull request merged**